packages/menu/src/SubMenu.tsx

```diff
@@ -325,9 +232,31 @@ const SubMenu: ExtendableBox<
 
     return (
       <li role="none" className={liStyle}>
-        {renderBox}
+        <MenuItem
+          {...rest}
+          {...subMenuContainer.prop}
+          aria-haspopup={true}
+          onClick={onRootClick}
+          ref={ref}
+          className={cx(
+            subMenuStyle,
+            {
+              [subMenuOpenStyle]: open,
+            },
+            className,
+          )}
+          disabled={disabled}
+          active={active}
+          size={'default'}
```

Comment:
Nit: This should probably either use the size enum for the component, or just be a string (`size='default'`).